src/components/Home.js

```diff
@@ -59,8 +61,8 @@ class Home extends Component {
     // TODO:add activeClassName for when the link is selected
     return (
       <Route exact path="/" render={() => (
-          this.isLoggedIn ? (
-            <Sidebar/>
+          this.state.isLoggedIn ? (
+            <Dashboard />
           ) : (
             <Redirect to="/login"/>
```

Comment:
I don't think redirect works at home component.
Please make those change `            this.props.history.push('/login')`

Reply:
replace `<Redirect to="/login"/>` with  `           this.props.history.push('/login')`

Reply:
This is a bug. And we identified bugs on the login url

Reply:
We will need to fix this before our next demo... creating a new branch `Fix-Login`

Reply:
This is not how the structure of components work.  Please see the Frontend Component Architecture document.